packages/react-devtools-shared/src/utils.js

```diff
@@ -437,6 +438,8 @@ export function getDataType(data: Object): DataType {
         return 'array_buffer';
       } else if (typeof data[Symbol.iterator] === 'function') {
         return 'iterator';
+      } else if (data[Symbol.iterator] === 'data') {
```

Comment:
This doesn't look right. Why would `data[Symbol.iterator]` return the string "data"?